Updated room-invitation section (#1265)
wechaty/wechaty.js.org: docusaurus/docs/api/room-invitation.md

---
title: Room Invitation
---

## Room Invitation

> Room Invitation is a  global class that accepts room invitation.This section gives a description of the methods of the Room Invitation class.

Reviewer: ```suggestion
Room Invitation is a global class that accepts room invitation. This section describes the methods of the Room Invitation class.
```

| Instance Methods | Return type |
|----------------|---------------|
| accept\(\)     | `Promise`   |
| inviter\(\)    | `Promise`(Contact)   |
| topic\(\)    | `Promise` (String)   |
| date\(\)    | `Promise` (Date)   |
| age\(\)    | `Promise` (Number)   |

### roomInvitation.accept\(\) ⇒ `Promise <void>`

> This method accepts the room invitation.See the following example:

Reviewer: ```suggestion
This method accepts the room invitation. See the following example:
```

### roomInvitation.inviter\(\) ⇒ `Promise <Contact>`

> This method gets the inviter from the room invitation. Check the following example below :

Reviewer: ```suggestion
This method gets the inviter from the room invitation. Check the following example:
```
